cmd/daemon_logger/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"flag"
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/Sirupsen/logrus"
+)
+
+var (
+	hostport  = flag.String("hostport", "", "Hostname and port of destination.")
+	namespace = flag.String("ns", "", "Namespace for metrics. Ex: daemontools")
+	name      = flag.String("name", "", "Name of metric to report. Ex: service.starts")
+	gauge     = flag.Float64("gauge", 0, "Report a 'gauge' metric. Value must be float64.")
+	timing    = flag.Duration("timing", time.Now().Sub(time.Now()), "Report a 'timing' metric. Value must be parseable by time.ParseDuration.")
+	timeinms  = flag.Float64("timeinms", 0, "Report a 'timing' metric, in milliseconds. Value must be float64.")
+	incr      = flag.Bool("incr", false, "Report an 'incr' metric.")
+	decr      = flag.Bool("decr", false, "Report a 'decr' metric.")
+	count     = flag.Int64("count", 0, "Report a 'count' metric. Value must be an integer.")
+	tag       = flag.String("tag", "", "Tag for metric. Ex: `service:airflow")
```

Comment:
I think this tool should support setting multiple tags for a metric (e.g., a service tag as well as a tag for a file path or something). Since golang's `flag` doesn't support passing the same flag twice, I guess the easiest way to do it would be to split tags by `,` (since the comma is conveniently disallowed in tags anyway).